MAUI/Rotator/Adding-Looping-and-Delays.md

```diff
@@ -0,0 +1,376 @@
+---
+layout: post
+title: Adding Looping and Delays in .NET MAUI Rotator control | Syncfusion
+description: Learn about Adding Looping and Delay support in Syncfusion .NET MAUI Rotator (SfRotator) control and more.
+platform: maui 
+control: Rotator
+documentation: ug
+---
+
+# Adding Looping and Delays in .NET MAUI Rotator (SfRotator)
+
+Looping and delay can be enabled in the `SfRotator` control, and you can customize the Text and Navigation direction.
```

Comment:
Recheck this sentence

Reply:
I had changed like Looping and delay can be enabled in the `SfRotator` control in .NET MAUI.